extensions/asde-deployment/package.nls.json

```diff
@@ -26,5 +26,18 @@
 	"remote-info-section-title": "Target machine information",
 	"edge-remote-target-field": "Name or IP address",
 	"edge-remote-username-field": "Username",
-	"edge-remote-password-field": "Password"
+	"edge-remote-password-field": "Password",
+	"sql-edge-azure-display-name": "An Azure VM via Azure IoT Hub",
+	"sql-edge-azure-notebook": "./notebooks/edge/deploy-sql-edge-azure.ipynb",
```

Comment:
Why are these paths in the localized file? I wouldn't expect these to ever be actually localized right?

Reply:
i guess you are right, i will move this to package.json to avoid confusing the localization team.